dev-packages/application-manager/src/generator/frontend-generator.ts

```diff
@@ -188,17 +188,22 @@ app.on('ready', () => {
             width, height, x, y
         });
 
-        let windowOptions = {
-            show: false,
-            title: applicationName,
-            width: windowState.width,
-            height: windowState.height,
-            minWidth: 200,
-            minHeight: 120,
-            x: windowState.x,
-            y: windowState.y,
-            isMaximized: windowState.isMaximized
-        };
+        let windowOptions = electronStore.get('windowOptions');
+
+        if (!windowOptions) {
+            windowOptions = {
+                show: false,
+                title: applicationName,
+                width: windowState.width,
+                height: windowState.height,
+                minWidth: 200,
+                minHeight: 120,
+                x: windowState.x,
+                y: windowState.y,
+                isMaximized: windowState.isMaximized
+            };
+            electronStore.set('windowOptions', windowOptions);
```

Comment:
I can't see how this would affect existing users, but should there be a flag whether to persist this?

Reply:
The standard's get persisted because the user can get them through this:
```
ipcMain.on('get-window-options', event => {
  event.returnValue = electronStore.get('windowOptions');
});
```
This allows the client to add something instead of replacing the defaults
Adding:
```
const currentWindowOptions = ipcRenderer.sendSync('get-window-options');
ipcRenderer.send('set-window-options', {
  ...currentWindowOptions,
  frame: false
});
```
The defaults could also be discarded and replaced with something completely new by the client.
Replacing:
```
ipcRenderer.send('set-window-options', {
  show: true,
  title: 'test',
  width: 20,
  height: 20,
  x: 0,
  y: 0
});
```

Reply:
I see, this could also be used to persist the user's last state I guess?